xskillscore/tests/test_probabilistic.py

```diff
@@ -423,7 +427,13 @@ def test_reliability_values(o, f_prob):
 def test_reliability_perfect_values(o):
     """Test values for perfect forecast"""
     f_prob = xr.concat(10 * [o], dim="member")
-    actual = reliability(o > 0.5, (f_prob > 0.5).mean("member"))
+    # scipy bins are only left-edge inclusive and 1e-8 is added to the last bin, whereas
+    # xhistogram the rightmost edge of xhistogram bins is included - mimic scipy behaviour
```

Comment:
Do you use scipy here? Isn’t only in the test above?

Reply:
Why is the eps needed here?

Reply:
We compare to scipy metrics in both `test_reliability_perfect_values` and `test_reliability_values`. In these tests, we now need to add an eps to our last bin to mimic how the bins are defined in the scipy functions (ie, all left edge inclusive only, but eps is add to the right edge of the last bin).

Reply:
I understand that, but this test only uses `reliability` from `xs` and not `scipy`. thats were I stumbled

Reply:
Oh, yeah - good point and good catch. I stupidly just assumed that this test failed for the same reason as `test_reliability_values`. But something else is happening here.

I'm not at a computer right now, but I'll try sort this asap.

Reply:
`reliability` does it's own binning (i.e. doesn't use xhistogram) and was set up to mimic the _old_ xhistogram behaviour. I've now changed this to follow the new behaviour.

Thanks for catching this @aaronspring!